tests/unit_tests/simtel/test_simtel_histograms.py

```diff
@@ -11,19 +11,23 @@
 def test_histograms(io_handler):
     histogram_files = list()
     histogram_files.append(
-        io_handler.getInputDataFile(
-            fileName="run1_gamma_za20deg_azm0deg-North-Prod5_test-production-5.hdata.zst", test=True
+        io_handler.get_input_data_file(
+            file_name="run1_gamma_za20deg_azm0deg-North-Prod5_test-production-5.hdata.zst",
+            test=True,
         )
     )
     histogram_files.append(
-        io_handler.getInputDataFile(
-            fileName="run2_gamma_za20deg_azm0deg-North-Prod5_test-production-5.hdata.zst", test=True
+        io_handler.get_input_data_file(
+            file_name="run2_gamma_za20deg_azm0deg-North-Prod5_test-production-5.hdata.zst",
+            test=True,
         )
     )
 
-    hists = SimtelHistograms(histogramFiles=histogram_files, test=True)
+    hists = SimtelHistograms(histogram_files=histogram_files, test=True)
 
-    figName = io_handler.getOutputFile(fileName="simtelHistograms.pdf", dirType="plots", test=True)
-    hists.plotAndSaveFigures(figName=figName)
+    fig_name = io_handler.get_output_file(
+        file_name="simtel_histograms.pdf", dir_type="plots", test=True
```

Comment:
You changed the name of the file here but I couldnt understand where this file is generated. Please make sure we are still going to find this file (I assume yes, since the test hasnt failed).

Reply:
The file is created in that same test in the next line so there's no issue. There are cases in which I changed the names of files in the code and also changed the files themselves.